### PR TITLE
start adding appInsigth client/server telemetry 

### PR DESCRIPTION
**Pull request by dacolgit**

### CODE REVIEW

File: samples/src/Microsoft.Azure.IIoT.App/src/Program.cs [hunk 1]
```diff
@@ -56,7 +57,10 @@ public class Program {
                     .UseKestrel(o => o.AddServerHeader = false)
                     .UseIISIntegration()
                     .UseSetting(WebHostDefaults.DetailedErrorsKey, "true"))
-                .UseSerilog();
+                .UseSerilog()
+                .ConfigureLogging(logging =>
+                    logging.AddFilter<Microsoft.Extensions.Logging.ApplicationInsights.ApplicationInsightsLoggerProvider>
```
Comment (marcschier) [inline samples/src/Microsoft.Azure.IIoT.App/src/Program.cs:62]: Add namespace to using and sort usings

File: samples/src/Microsoft.Azure.IIoT.App/src/Connected Services/Application Insights/ConnectedService.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿{
```
Comment (marcschier) [inline samples/src/Microsoft.Azure.IIoT.App/src/Connected Services/Application Insights/ConnectedService.json:1]: Is this file needed?  If not, please remove.

File: samples/src/Microsoft.Azure.IIoT.App/src/Program.cs (hunk 1 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Program.cs:62]: done

File: samples/src/Microsoft.Azure.IIoT.App/src/Connected Services/Application Insights/ConnectedService.json (hunk 2 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Connected Services/Application Insights/ConnectedService.json:1]: I checked this file is simply a marker to the Visual Studio "Connected Services" tooling that Application Insights was installed. Removed
